test/test_binary_ufuncs.py

```diff
@@ -2684,6 +2684,12 @@ def _test_atan2_with_size(size, device):
                                     device=device, dtype=torch.double)
             self.assertEqual(expected, actual.view(-1), rtol=0, atol=0.02)
 
+            # bfloat16
+            a_bf16 = a.bfloat16()
+            b_bf16 = b.bfloat16()
+            actual_bf16 = a_bf16.atan2(b_bf16)
+            self.assertEqual(actual_bf16, actual.bfloat16())
```

Comment:
I think we'd want to check against expected, since `actual` is similar to `expected` within certain tolerance only.

Reply:
Also you can do `self.assertEqual(actual_bf16, expected, exact_dtype=False, rtol=,atol=)`.

Reply:
I think we can just remove this `self.assertEqual(actual_bf16, actual.bfloat16())` as we are testing with expected below.